src/js/localstorage.js

```diff
@@ -0,0 +1,187 @@
+    score: 0,        // increase by 1 after correct answer
+    lives: 3,        // decrease by 1 after wrong answer
+    };
+saveToLocalStorage(player,"player");
+}
+
+export const getPlayer = () => {
+    return loadFromLocalStorage("player");
+};
+
+//============================================================
+
+export const setScoreboard = () => {
+    const scoreboard = [{name: '', score: 0}];
+
+saveToLocalStorage(scoreboard, "scoreboard");
+}
+
+const getScoreboard = () => {
+    return loadFromLocalStorage("scoreboard");
+};
+
+//API PARTICULAR FUNCTIONS
+
+const setQuestions = function (...ques){ //ques == questions, could be any type
+    const game = getGame();
+    localStorage.removeItem("game"); // to prevent bugs
+    game.questions = ques;
+    saveToLocalStorage(game, "game");
+    console.log(ques);
+};
+
+
+const nextQuestion = function () {
+    const game = getGame();
+    localStorage.removeItem("game");
+    game.currentQuestion++;
+    saveToLocalStorage(game, "game");
+};
+
+const setPath = function (p) {
+   const game = getGame();
+   localStorage.removeItem("game"); 
+   game.path = p;
+   saveToLocalStorage(game, "game");
+};
+
+const setTier = function (t) {
+    const game = getGame();
+    localStorage.removeItem("game");
+    game.tier = t;
+    saveToLocalStorage(game, "game");
+};
+
+const setPlayerName = function (n) {
+    const player = getPlayer();
+    localStorage.removeItem("player");
+    player.name = n;
+    saveToLocalStorage(player, "player");
+};
+
+const useSkip = function () {
+    const player = getPlayer();
+    localStorage.removeItem("player");
+    player.skipUsed = true;
+    saveToLocalStorage(player, "player");
+};
+
+const addPoint = function () {
+    const player = getPlayer();
+    localStorage.removeItem("player");
+    player.score++;
+    saveToLocalStorage(player, "player");
+};
+
+const removeLife = function () {
+    const player = getPlayer();
+    localStorage.removeItem("player");
+    player.lives--;
+    saveToLocalStorage(player, "player");
+};
+
+const addScore = function (n, s) { //n - string, player name, s - number, score
+    const objToPush = {
+        name: n,
+        score: s,
+    };
+    const scoreboard = getScoreboard();
+    localStorage.removeItem("scoreboard");
+    scoreboard.push(objToPush);
+    saveToLocalStorage(scoreboard, "scoreboard");
+
+};
+
+// OBJECT TO IMPORT ALL FUNCTIONS 
+
+export const ls = {                             
+    localStorageTest: localStorageTest,
+    lockStart: lockStart,
+    saveToLocalStorage: saveToLocalStorage,
+    loadFromLocalStorage: loadFromLocalStorage,
+    getPlayer: getPlayer,
+    getGame: getGame,
+    getScoreboard: getScoreboard,
+    setQuestions: setQuestions,
+    nextQuestion: nextQuestion,
+    setPath: setPath,
+    setTier: setTier,
+    setPlayerName: setPlayerName,
+    useSkip: useSkip,
+    addPoint: addPoint,
+    removeLife: removeLife,
+    addScore: addScore
+};
```

Comment:
We can export variables right before their definition.
`export const someVariable = ....`

Reply:
We decided to leave it like this but in the shorter version.